### ISSUE TITLE
Controller blocked being connected directly to stream PC

### ISSUE THREAD

**Issue by prounlocks** [moonlight-stream/moonlight-android#1047]
PC Windows 11 PRO 21H2   GF experience 3.25.0.84 streaming to
Shield 2019 PRO
XBOX ONE S  1914 model controller connected via BT directly to PC windows.
Launching game by open desktop (explorer.exe in moonlight interface), after manually loading game.
Other words - moonlight works like image transfer only.
All moonlight controller supports are disabled (tried with enabled too)

I found strange bug in two games (rest looks good):

1) Star wars Jedi  Fallen orders -  only left analogue stick works - nothing more
2) God Of War - nothing works (controller blocked)

Same games without streaming working good with same controller.

This issue happens not only Moonlight, but with original Nvidia shield streaming client too.

First time i thought that game using my BT Shield remote control as a promary controller, but i disconnected it, reconnect streaming,  nothing changes.

So looks like XINPUT works different during streaming and prevents to use controllers connected to PC directly.

If the controller connected to Nvidia shield - no problems.

**Comment by cgutman** [comment]
Yeah, this is something that Nvidia does, though it has gone back and forth over the years as they adjusted their controller emulation code.

I think their decision makes sense of preferring remote controllers makes sense, because the alternative is that games that only enumerate one controller would be totally unplayable via GameStream if you left a controller connected to your PC (as many people do).

Since this is not a Moonlight bug/behavior, I will transfer this to the other issue tracker.
